src/applications/personalization/profile-2/tests/e2e/profile.personal-and-contact-information-modals.cypress.spec.js

```diff
@@ -0,0 +1,176 @@
+import { PROFILE_PATHS } from '../../constants';
+
+import mockUser from '../fixtures/users/user-36.json';
+import mockFeatureToggles from '../fixtures/feature-toggles.json';
+
+const setup = (mobile = false) => {
+  window.localStorage.setItem(
+    'DISMISSED_ANNOUNCEMENTS',
+    JSON.stringify(['single-sign-on-intro']),
+  );
+
+  if (mobile) {
+    cy.viewport('iphone-4');
+  }
+
+  cy.login(mockUser);
+  cy.route('GET', '/v0/feature_toggles*', mockFeatureToggles);
+  cy.visit(PROFILE_PATHS.PROFILE_ROOT);
+
+  // should show a loading indicator
+  cy.findByRole('progressbar').should('exist');
```

Comment:
Although I doubt this is slowing anything down, I think we've done these checks enough that we could probably skip them? I don't feel super strongly about it either way.

Reply:
Although probably not ideal to use them for this purpose, I believe they provide a good buffer to have the page load so that the test can find the next elements. Without adding these tests, I can only reliably have those tests run successfully (locally) by either adding a `cy.wait(5000)` or increasing the default 4000 timeout in `cypress.config` (`defaultCommandTimeout`) to a higher value, which we might want to avoid.. if ok with you, I will leave those in for now.

Reply:
Yeah leaving them is totally fine with me. I don't think the longer timeout is needed now, though, since you are using `cy` to mock all the endpoints when running locally. I think the fake API calls that run locally were the problem before.